templates/single-product/add-to-cart/variable.php

```diff
@@ -35,7 +35,7 @@
 			<tbody>
 				<?php foreach ( $attributes as $attribute_name => $options ) : ?>
 					<tr>
-						<td class="label"><label for="<?php echo esc_attr( sanitize_title( $attribute_name ) ); ?>"><?php echo wc_attribute_label( $attribute_name ); // WPCS: XSS ok. ?></label></td>
+						<td class="label"><label for="<?php echo esc_attr( 'attribute_' . $product->get_id() . '_' . sanitize_title( $attribute_name ) ); ?>"><?php echo wc_attribute_label( $attribute_name ); // WPCS: XSS ok. ?></label></td>
```

Comment:
can you generate the id in a variable and pass it in the parameter for `wc_dropdown_variation_attribute_options`. also can you pass the attribute_name as a class so that anyone targeting id just need to switch from id to class.